### PR TITLE
Added pin to user model with front end updation 

### PR DESCRIPTION
**Pull request by sukhbir-singh**

### CODE REVIEW

**Comment by csoni111** [comment]
@sukhbir-singh Please also add a check so that no two users can set same pin.

**Comment by sukhbir-singh** [comment]
okay!! I will do the changes

File: app/models/user.rb [hunk 1]
```diff
@@ -45,6 +45,9 @@ class User < ApplicationRecord
 
 	validates :name, :presence => true
 
+	validate :validate_pin
```
Comment (cpg) [inline app/models/user.rb]: you could more succintly do validate :validate_pin, unless: pin.empty?

Comment (sukhbir-singh) [reply app/models/user.rb]: yes, this will now reduce the check in validator. Thanks !!

**Pull request merged**